### PR TITLE
refactor: move entry points to testbench module

### PR DESCRIPTION
**Pull request by coryan**
This is part of the effort to remove any references to "emulator" in the
repository. I left `emulator.py` around for backwards compatibility, but
I suspect we can remove it after checking with a few folks.

Part of the work for #27 

The line count looks scary: I renamed `emulator.py` to `testbench/rest_server.py`, and then added some small functions for backwards compatibility.

### CODE REVIEW

File: testbench/__main__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
+#
+# Copyright 2020 Google LLC
```
Comment (cojenco) [inline testbench/__main__.py:3]: Thanks for the thorough scrub and move! 
Just one nit and q: Did we ever decide on keeping the original date or updating to 2021 for changes like these?

File: testbench/rest_server.py [hunk 2]
```diff
@@ -0,0 +1,897 @@
+# Copyright 2020 Google LLC
```
Comment (cojenco) [inline testbench/rest_server.py:1]: Same question here

Comment (coryan) [reply testbench/rest_server.py]: Fixed.  I thought `git` was going to treat that as a move, but I never argue with the robots. Fixed, and reverted the change in `emulator.py`.

File: testbench/__main__.py (hunk 1 above)
Comment (coryan) [reply testbench/__main__.py]: This is a mistake, fixed.

**Comment by coryan** [comment]
Rebased, resolved conflicts and addressed your comments. PTAL.

**Review by cojenco — APPROVED** [review]
LGTM👍 Thanks Carlos!

**Pull request merged**